collection_requirements.rst

```diff
@@ -191,6 +191,18 @@ Also:
 See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ for complete details.
 
 
+List of requirements for collections to be included in Ansible
+==============================================================
+
+To be included in the `ansible` package, collections must meet the following criteria:
+
+* `development conventions <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html>`_
+* `Collection requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ (this document)
+* `Ansible documentation format <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html>`_ and the `style guide <https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#style-guide>`_
+* to pass the Ansible `sanity tests <https://docs.ansible.com/ansible/devel/dev_guide/testing_sanity.html#testing-sanity>`_
+* to have `unit <https://docs.ansible.com/ansible/devel/dev_guide/testing_units.html#testing-units>`_ and / or `integration tests <https://docs.ansible.com/ansible/devel/dev_guide/testing_integration.html#testing-integration>`_
```

Comment:
when I click on the links, they take me directly to the content and just *below* the header, would it be better to use the header links for maybe a bit more context when landing on the page?

```suggestion
* to have `unit <https://docs.ansible.com/ansible/devel/dev_guide/testing_units.html#unit-tests>`_ and / or `integration tests <https://docs.ansible.com/ansible/devel/dev_guide/testing_integration.html#integration-tests>`_
```

Reply:
@dericcrago ok, sounds good, thanks!

Reply:
committed